en/flight_modes/takeoff.md

```diff
@@ -31,47 +31,83 @@ Parameter | Description
+
+There are generally two ways to start an automatic takeoff on fixed-wing vehicles: either by planning a mission takeoff and starting the mission, or by switching to the Takeoff flight mode and arm the vehicle.
+Both methods lead to very similar behavior, with the difference being that in a mission takeoff, the user can specify the takeoff course (vehicle will track this course after being clear of the runway) and takeoff altitude (vehicle will climb to this altitude while staying on the takeoff course).
+For the non-mission takeoff, the course is set to the vehicle heading on arming, and the altitude is set to [MIS_TAKEOFF_ALT](#MIS_TAKEOFF_ALT).
+
+
+Parameters that apply to both catapult/hand-launch as well as runway takeoffs:
+
+Parameter | Description
+--- | ---
+<span id="MIS_TAKEOFF_ALT"></span>[MIS_TAKEOFF_ALT](../advanced_config/parameter_reference.md#MIS_TAKEOFF_ALT) | Altitude setpoint above Home that the vehicle will climb to during takeoff if not in mission mode.
+<span id="FW_TKO_AIRSPD"></span>[FW_TKO_AIRSPD](../advanced_config/parameter_reference.md#FW_TKO_AIRSPD) | Takeoff airspeed (is set to [FW_AIRSPD_MIN](../advanced_config/parameter_reference.md#FW_AIRSPD_MIN) if not defined by operator)
+<span id="FW_TKO_PITCH_MIN"></span>[FW_TKO_PITCH_MIN](../advanced_config/parameter_reference.md#FW_TKO_PITCH_MIN) | This is the minimum pitch angle setpoint during the climbout phase.
+<span id="FW_THR_MAX"></span>[FW_THR_MAX](../advanced_config/parameter_reference.md#FW_THR_MAX) | Maximum throttle during the whole takeoff phase.
+<span id="FW_T_CLMB_MAX"></span>[FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX) | Climb rate setpoint during climbout phase.
+
+
 
-The aircraft takes off in the current direction using either *catapult/hand-launch mode* or *runway takeoff mode*.
-The mode defaults to catapult/hand launch, but can be set to runway takeoff using [RWTO_TKOFF](#RWTO_TKOFF).
-RC stick movement is ignored in both cases.
 
 <span id="hand_launch"></span>
 ### Catapult/Hand Launch
 
-In *catapult/hand launch mode* the vehicle waits to detect launch (based on acceleration trigger).
-On launch it ramps up to full throttle ([RWTO_MAX_THR](#RWTO_MAX_THR)) in about 2 seconds and then performs a full throttle climbout, with *minimum* 10 degree takeoff pitch. 
-Once it reaches [FW_CLMBOUT_DIFF](#FW_CLMBOUT_DIFF) it will transition to [Hold mode](../flight_modes/hold.md) and loiter.
+In *catapult/hand-launch mode* the vehicle waits to detect launch (based on acceleration trigger).
+On launch it ramps up to full throttle [FW_THR_MAX](#FW_THR_MAX) and climbs with the maximum climb rate [FW_T_CLMB_MAX](#FW_T_CLMB_MAX) while keeping the pitch setpoint above [FW_TKO_PITCH_MIN](#FW_TKO_PITCH_MIN).
+Once it reaches [MIS_TAKEOFF_ALT](#MIS_TAKEOFF_ALT) (or altitude defined in the mission item) it will automatically switch to [Hold mode](../flight_modes/hold.md) and loiter.

```

Comment:
This is the takeoff mode doc - if you have that caveat this is incorrect because in a mission it will continue to the next mission item before continuing.
I think if you can have a catapult launch in a mission mode, we need to have that information in the mission mode doc or "below" it. 

```suggestion
Once it reaches [MIS_TAKEOFF_ALT](#MIS_TAKEOFF_ALT) it will automatically switch to [Hold mode](../flight_modes/hold.md) and loiter.

```

Reply:
Yes this should be cleanly separated: Loiter in case of mission-less takeoff, proceed to next WP in Mission.

Reply:
there's also some repeat info in the paragraph from before and this one. maybe could do another revision pass through and even just describe each mode separately (with less general description beforehand)

Reply:
removed reference to the mission item altitude here.